### PR TITLE
Onboarding Highlights

### PR DESCRIPTION
**Pull request by mgurgel**
Asana: https://app.asana.com/0/1206594217596623/1208204606544051/f

Onboarding Highlights flow for Mac and Windows

[Figma](https://www.figma.com/design/s9cFusskywDt9NFNFuqRcG/Onboarding---Highlights?node-id=7344-15538)
[Prototype](https://www.figma.com/proto/4Iqc3ADrRDcZgBbQJwxaPQ/Highlights-(Desktop)---E.10?page-id=0%3A1&node-id=1-32513&node-type=frame&viewport=2234%2C-4081%2C1.09&t=qaSIEawZATbGVaCf-8&scaling=contain&content-scaling=responsive&starting-point-node-id=1%3A32513&disable-default-keyboard-nav=1&hotspot-hints=0&hide-ui=1)

Previews:
- [Mac](https://deploy-preview-1059--content-scope-scripts.netlify.app/build/pages/onboarding/?platform=macos)
- [Windows](https://deploy-preview-1059--content-scope-scripts.netlify.app/build/pages/onboarding/?platform=windows)

### CODE REVIEW

File: packages/special-pages/pages/onboarding/app/settings.js [hunk 1]
```diff
@@ -7,23 +7,41 @@ import { stepDefinitions as defaultStepDefinitions } from './data'
...
     } = {}) {
+        this.platform = platform
         this.order = order
+        this.orderName = orderName
         this.stepDefinitions = stepDefinitions
         this.first = first
         this.exclude = exclude
     }
 
+    withPlatformName (name) {
```
Comment (mgurgel) [inline packages/special-pages/pages/onboarding/app/settings.js:33]: Setting up for future implementation of platformName

File: special-pages/pages/onboarding/app/components/Typed.js [hunk 2]
```diff
@@ -72,10 +74,26 @@ function TypedInner ({ text, onComplete, delay, children, ...rest }) {
...
 
     useEffect(() => {
+        if (paused) return () => {}
+
         const controller = new AbortController()
         let enabled = true
 
-        document.body.addEventListener('pointerdown', () => {
+        document.body.addEventListener('pointerdown', (e) => {
+            // TODO: Check that it's not affecting v1/v2
```
Comment (shakyShane) [inline special-pages/pages/onboarding/app/components/Typed.js:83]: remove this if you've verified it - just to be clear it's no-longer an issue.

File: special-pages/pages/onboarding/app/global.js [hunk 3]
```diff
@@ -17,10 +17,10 @@ export const GlobalDispatch = createContext(/** @type {import("preact/hooks").Di
...
  */
 export function reducer (state, action) {
-    // console.group('reducer');
-    // console.log('state', state);
-    // console.log('action', action);
-    // console.groupEnd()
+    console.group('reducer')
+    console.log('state', state)
+    console.log('action', action)
+    console.groupEnd()
```
Comment (shakyShane) [inline special-pages/pages/onboarding/app/global.js:23]: let's remove these again to prevent logging noise in prod?

**Review by shakyShane — CHANGES_REQUESTED** [review]

**Review by shakyShane — APPROVED** [review]
Tested extensively for weeks with different teams. 

I also validated that this new work does not impact the existing flows on windows (v1+v2)

**Pull request merged**